src/ContentUtils.elm

```diff
@@ -5,6 +5,7 @@ import List
 import Pages
 import Date.Extra
 import Posts
+import String exposing (contains, toLower)
```

Comment:
final thing - don't `exposing (contains, toLower)` here - just `import String`. Then call `String.contains` and `String.toLower`. That way it's clearer way the functions are coming from.